docs/tutorials/neutron-diffraction.ipynb

```diff
@@ -401,13 +401,14 @@
     "                    values=np.linspace(0.0, np.pi/2, num=16))\n",
     "\n",
     "# After conversion to d-spacing, bins of different spectra are not aligned.\n",
-    "# Therefore we first need to concatenate bins along d-spacing before concatenateing spectra.\n",
+    "# Therefore we first need to concatenate bins along d-spacing before concatenating spectra.\n",
     "tmp = dspacing_sample.bins.concatenate('d-spacing')\n",
     "focussed_sample = sc.groupby(tmp, 'theta', bins=theta).bins.concatenate('spectrum')\n",
     "tmp = dspacing_vanadium.bins.concatenate('d-spacing')\n",
     "focussed_vanadium = sc.groupby(tmp, 'theta', bins=theta).bins.concatenate('spectrum')\n",
-    "focussed_sample.bins.groupby['d-spacing'] /= sc.histogram(focussed_vanadium, dspacing_bins)\n",
-    "normalized = sc.histogram(focussed_sample, dspacing_bins)"
+    "norm = sc.histogram(focussed_vanadium.bins, dspacing_bins)\n",
+    "focussed_sample.bins /= sc.lookup(func=norm, dim='d-spacing')\n",
```

Comment:
If monitors were run in event mode would it be necessary to histogram them before this lookup approach was possible? While histogram monitors is usually what is required, there are legitimate cases where monitors are recorded in event mode.

Reply:
Yes, see https://scipp.github.io/event-data/overview.html#Arithmetic-operations. Only addition and subtraction are supported if both operands are event data.